components/description-list/description-list-wrapper.js

```diff
@@ -64,7 +64,8 @@ class DescriptionListWrapper extends LitElement {
 
 	constructor() {
 		super();
-		this.breakpoint = 200;
+		// TODO: can this be REM?
```

Comment:
The value it's compared to is in pixels. Accepting a rem value would require converting it using root font-size that would have to be obtained from the document. Looks like our other components just accept pixels.

Reply:
Yeah, this was mostly a todo for myself next week to see if this was possible/feasible. Seems like PX is the convention (and what makes sense here).